core/storage/story/gae_models.py

```diff
@@ -70,6 +70,8 @@ class StoryModel(base_models.VersionedModel):
         ndb.IntegerProperty(required=True, indexed=True))
     # The topic id to which the story belongs.
     corresponding_topic_id = ndb.StringProperty(indexed=True, required=True)
+    # The url fragment for the story.
+    url_fragment = ndb.StringProperty(indexed=True)
```

Comment:
Would it be required? Or do we need to do that later on (if so, file a TODO/issue)